bsd.go

```diff
@@ -7,23 +7,23 @@ package gopass
 #include <unistd.h>
 #include <stdio.h>
 
-int getch() {
-        int ch;
+char getch(int termDescriptor) {
+        char ch;
         struct termios t_old, t_new;
 
-        tcgetattr(STDIN_FILENO, &t_old);
+        tcgetattr(termDescriptor, &t_old);
         t_new = t_old;
         t_new.c_lflag &= ~(ICANON | ECHO);
-        tcsetattr(STDIN_FILENO, TCSANOW, &t_new);
+        tcsetattr(termDescriptor, TCSANOW, &t_new);
 
-        ch = getchar();
+        read(termDescriptor, &ch, sizeof(ch));
```

Comment:
i'm not a fan of this change.  It's customary for `getch` to return int, since it needs to return `-1` on EOF. Some platforms have signed chars, and some have unsigned chars, so you can't count on `char` to convey `<0`

What's the argument against leaving `ch = getchar()` as is?